python/pyhie/allspark/model/llama_v4.py

```
@@ -658,6 +727,9 @@ def _load_expert(name):
             if quantize_mode is False:
                 save_torch_to_allsparky(weights_path, key, tensor, mode,
                                         split_mode, group_list)
+                if is_fp8 and scale_inv is not None:
+                    save_torch_to_allsparky(weights_path, key + ".scale_inv",
```

Flake8 (7.3.0)

[error] 'save_torch_to_allsparky' may be undefined, or defined from star imports: ..quantization, .model_base, .quantization_utils

(F405)